src/components/DashboardLayout/SettingsLeftNav.tsx

```diff
@@ -1,51 +1,110 @@
-import { useContext } from 'react';
+import { useContext, useState, useEffect } from 'react';
 import Button from 'components/Atoms/Button';
-import { faXmark } from '@fortawesome/free-solid-svg-icons';
+import { faXmark, faServer, faRightToBracket } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { BrowserLocationContext } from 'data/BrowserLocationContext';
 import { RadioGroup } from '@headlessui/react';
 import { SettingsContext } from 'data/SettingsContext';
 import Avatar from 'components/Atoms/Avatar';
 import clsx from 'clsx';
 import { useUid } from 'data/UserInfo';
+import CoreSettings from 'components/Settings/CoreSettings';
+import UserSettings from 'components/Settings/UserSettings';
+
+
+export const tabList = [
```

Comment:
I don't think this is the best place to put the tablist
It might be better to find another place for it?

Reply:
Maybe just put it in `index.tsx`